english/strange-room/room.md

```diff
@@ -4,7 +4,9 @@ It's obvious someone has been here recently but who?
 
 You don't feel scared in the room for now, so you decide to look around for a
 few minutes. You look under the table, and find a card with a random phone
-number written on it.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.